src/views/v2/components/events-bar/search-button/icon.php

```diff
@@ -13,4 +13,4 @@
  *
  */
 ?>
-<span class="tribe-events-c-events-bar__search-button-icon tribe-common-svgicon"></span>
+<?php $this->template('components/icons/search', [ 'classes' => ['tribe-events-c-events-bar__search-button-icon-svg'] ] ) ?>
```

Comment:
* :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space before array closer.
* :no_entry_sign: Missing space before array closer.
* :no_entry_sign: A closing tag is not permitted at the end of a PHP file

<sub>hash: 3ad3c45269e511918118573019bb7fcc</sub>